pyproject.toml

```diff
@@ -15,7 +15,7 @@ nanopb = "^0.4.3"
 
 ## test tools
 pytest = "^8"
-coverage = "<5"  # coverage 5+ uses binary sqlite format
+coverage = "5.5"  # coverage 5+ uses binary sqlite format
```

Comment:
no need to retain the comment now that we support the newer coverage format

Reply:
https://github.com/trezor/trezor-firmware/pull/4915/commits/fecf29ee55c64baa31158ca0ea2f2557b2f803de